### ISSUE TITLE
Failed run of pytorch_toolkit/object_detection

### ISSUE THREAD

**Issue by StianHanssen** [opencv/openvino_training_extensions#242]
**Describe the bug**
Get "missing keys in source state_dict: bbox_head.loss_weights" and "non-zero exit status 1" when following instructions on object_detection. Perhaps the incorrect checkpoint is provided from [step 3](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/person_vehicle_bike_detection.md)?

**Steps to Reproduce**

1. Follow install instructions from [here](https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/object_detection).
2. Follow further instructions from [here](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/person_vehicle_bike_detection.md).
3. Get error in step 4 by following:
```
To start training, run the following:

../../external/mmdetection/tools/dist_train.sh configs/person_vehicle_bike_sd512_mb2_clustered.py 1
```
From location 'openvino_training_extensions/pytorch_toolkit/object_detection' in terminal.

**Environment:**
 - OS: Linux Ubuntu 16.04.4
 - Framework version: PyTorch 1.1.0
 - Python version: Python 3.6.9
 - OpenVINO version: 2020.1.023
 - CUDA/cuDNN version: 10.1
 - GPU model and memory: GeForce RTX 2060 with 6GB GDDR6

**Terminal Output:**
```
(detection)stian@stian-P65:~/Projects/openvino_training_extensions/pytorch_toolkit/object_detection$ ../../external/mmdetection/tools/dist_train.sh configs/person_vehicle_bike_sd512_mb2_clustered.py 1
2020-03-15 10:16:01,377 - INFO - Distributed training: True
2020-03-15 10:16:03,381 - INFO - load checkpoint from ./checkpoint/person_vehicle_bike_sd512_mb2_clustered_epoch_21.pth
2020-03-15 10:16:03,401 - WARNING - unexpected key in source state_dict: backbone.extra_convs.4.0.weight, backbone.extra_convs.4.1.weight, backbone.extra_convs.4.1.bias, backbone.extra_convs.4.1.running_mean, backbone.extra_convs.4.1.running_var, backbone.extra_convs.4.1.num_batches_tracked, backbone.extra_convs.5.0.weight, backbone.extra_convs.5.1.weight, backbone.extra_convs.5.1.bias, backbone.extra_convs.5.1.running_mean, backbone.extra_convs.5.1.running_var, backbone.extra_convs.5.1.num_batches_tracked, backbone.extra_convs.6.0.weight, backbone.extra_convs.6.1.weight, backbone.extra_convs.6.1.bias, backbone.extra_convs.6.1.running_mean, backbone.extra_convs.6.1.running_var, backbone.extra_convs.6.1.num_batches_tracked, backbone.extra_convs.7.0.weight, backbone.extra_convs.7.1.weight, backbone.extra_convs.7.1.bias, backbone.extra_convs.7.1.running_mean, backbone.extra_convs.7.1.running_var, backbone.extra_convs.7.1.num_batches_tracked, backbone.extra_convs.8.0.weight, backbone.extra_convs.8.1.weight, backbone.extra_convs.8.1.bias, backbone.extra_convs.8.1.running_mean, backbone.extra_convs.8.1.running_var, backbone.extra_convs.8.1.num_batches_tracked, backbone.extra_convs.9.0.weight, backbone.extra_convs.9.1.weight, backbone.extra_convs.9.1.bias, backbone.extra_convs.9.1.running_mean, backbone.extra_convs.9.1.running_var, backbone.extra_convs.9.1.num_batches_tracked, backbone.extra_convs.10.0.weight, backbone.extra_convs.10.1.weight, backbone.extra_convs.10.1.bias, backbone.extra_convs.10.1.running_mean, backbone.extra_convs.10.1.running_var, backbone.extra_convs.10.1.num_batches_tracked, backbone.extra_convs.11.0.weight, backbone.extra_convs.11.1.weight, backbone.extra_convs.11.1.bias, backbone.extra_convs.11.1.running_mean, backbone.extra_convs.11.1.running_var, backbone.extra_convs.11.1.num_batches_tracked, backbone.extra_convs.12.0.weight, backbone.extra_convs.12.1.weight, backbone.extra_convs.12.1.bias, backbone.extra_convs.12.1.running_mean, backbone.extra_convs.12.1.running_var, backbone.extra_convs.12.1.num_batches_tracked, bbox_head.loss_balancer.weight

missing keys in source state_dict: bbox_head.loss_weights

2020-03-15 10:16:03,401 - INFO - Start running, host: stian@stian-P65, work_dir: /home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/models/person_vehicle_bike_sd512_mb2_clustered
2020-03-15 10:16:03,401 - INFO - workflow: [('train', 1)], max: 5 epochs
Traceback (most recent call last):
  File "../../external/mmdetection/tools/train.py", line 98, in <module>
    main()
  File "../../external/mmdetection/tools/train.py", line 94, in main
    logger=logger)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/apis/train.py", line 60, in train_detector
    _dist_train(model, dataset, cfg, validate=validate)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/apis/train.py", line 189, in _dist_train
    runner.run(data_loaders, cfg.workflow, cfg.total_epochs)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/mmcv/runner/runner.py", line 358, in run
    epoch_runner(data_loaders[i], **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/mmcv/runner/runner.py", line 264, in train
    self.model, data_batch, train_mode=True, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/apis/train.py", line 40, in batch_processor
    losses = model(**data)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/mmcv/parallel/distributed.py", line 50, in forward
    return self.module(*inputs[0], **kwargs[0])
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/core/fp16/decorators.py", line 49, in new_func
    return old_func(*args, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/models/detectors/base.py", line 86, in forward
    return self.forward_train(img, img_meta, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/models/detectors/single_stage.py", line 51, in forward_train
    x = self.extract_feat(img)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/models/detectors/single_stage.py", line 40, in extract_feat
    x = self.backbone(img)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/external/mmdetection/mmdet/models/backbones/mobilenetv2.py", line 226, in forward
    x = self.bn_first(x)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/modules/batchnorm.py", line 83, in forward
    exponential_average_factor, self.eps)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/nn/functional.py", line 1697, in batch_norm
    training, momentum, eps, torch.backends.cudnn.enabled
RuntimeError: cuDNN error: CUDNN_STATUS_EXECUTION_FAILED
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/distributed/launch.py", line 235, in <module>
    main()
  File "/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/lib/python3.6/site-packages/torch/distributed/launch.py", line 231, in main
    cmd=process.args)
subprocess.CalledProcessError: Command '['/home/stian/Projects/openvino_training_extensions/pytorch_toolkit/object_detection/venv/bin/python', '-u', '../../external/mmdetection/tools/train.py', '--local_rank=0', 'configs/person_vehicle_bike_sd512_mb2_clustered.py', '--launcher', 'pytorch']' returned non-zero exit status 1.
```

**Comment by StianHanssen** [comment]
After some experimenting I found that the RuntimeError: cuDNN error: CUDNN_STATUS_EXECUTION_FAILED was likely cause by some problems with my installation of Cuda Toolkit. By changing [init_venv.sh](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/object_detection/init_venv.sh) to use conda instead and installing pytorch, torchvision and Cuda Toolkit with conda, I got it running.

However, the checkpoint issue still remain , and I still believe it is due to a mismatch between the model in the project and the checkpoint data provided.
